package.json

```diff
@@ -3,29 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@emotion/react": "^11.10.5",
-    "@emotion/styled": "^11.10.5",
     "@mui/icons-material": "^5.11.0",
     "@mui/material": "^5.11.0",
-    "axios": "^0.27.2",
-    "jquery": "^3.7.0",
+    "axios": "^1.6.5",
     "lightbox2": "^2.11.3",
     "notistack": "^3.0.1",
     "react": "^18.0.2",
     "react-dom": "^18.0.2",
     "react-helmet": "^6.1.0",
-    "react-image-crop": "^10.1.5",
+    "react-image-crop": "^11.0.4",
     "react-infinite-scroller": "^1.2.6",
     "react-masonry-infinite": "^1.2.2",
     "react-router-dom": "^6.3.0",
-    "react-scripts": "5.0.1",
     "react-select": "^5.3.2",
     "react-select-search": "^3.0.10",
     "react-transition-group": "^4.4.2",
-    "reactstrap": "^9.1.1",
-    "styled-components": "^5.3.5",
-    "vite-raw-plugin": "^1.0.1",
-    "web-vitals": "^2.1.4"
+    "styled-components": "^6.1.8",
```

Comment:
If we want to use styled components, we should probably add @mui/styled-engine-sc.
Also, we should probably not use them at all, because we can achieve most of the stuff with the default material ui.

Reply:
We use styled components for https://github.com/icpc/gallery/blob/0c24e98f413c60982540fe3fb82947b93b3e71f8/src/components/Body/FaceDiv.jsx#L6
I think it's fine and no need for mui/styled-engine-sc considering everything works